script/site-specific-install.sh

```diff
@@ -34,7 +34,7 @@ clear_daemon() {
 
 install_daemon() {
   echo -n "Creating /etc/init.d/$SITE-$1... "
-  (su -l -c "cd '$REPOSITORY' && bundle exec rake config_files:convert_init_script DEPLOY_USER='$UNIX_USER' VHOST_DIR='$DIRECTORY' RUBY_VERSION='$RUBY_VERSION' SCRIPT_FILE=config/$1-debian.example" "$UNIX_USER") > /etc/init.d/"$SITE-$1"
+  (su -l -c "cd '$REPOSITORY' && bundle exec rake config_files:convert_init_script DEPLOY_USER='$UNIX_USER' VHOST_DIR='$DIRECTORY' VCSPATH='$SITE' SITE='$SITE' RUBY_VERSION='$RUBY_VERSION' USE_RBENV=$USE_RBENV RAILS_ENV='production' SCRIPT_FILE=config/$1-debian.example" "$UNIX_USER") > /etc/init.d/"$SITE-$1"
```

Comment:
Also `VCSPATH` and `SITE` on this line? How come these are needed now vs before? (I'm sure there's a good reason, but I don't understand from looking at the commit in isolation)

Reply:
I'm not saying we _shouldn't_ do it; I just think it should be explained (and its own commit is probably a better place to do that).

The addition of `RAILS_ENV=production` would resolve https://github.com/mysociety/alaveteli/issues/2392, and as far as I can tell, that's what we _should_ be doing. I do remember something confusing about all this, but can't find any specific reason why we shouldn't make this specific change. Maybe it was https://github.com/mysociety/alaveteli/pull/2242#issuecomment-85485657? That doesn't really suggest that we shouldn't make this change, since we only generate `/etc/init.d/$SITE` if we're _not_ creating a `$DEVELOPMENT_INSTALL`.

Reply:
I've split this change into 3 commits:
- 4a9601c with the new `USE_RBENV` variable,
- 7180f42 makes the variables consistent,
- 245833e adds the RAILS_ENV variable, now based on DEVELOPMENT_INSTALL